### ISSUE TITLE
Hippodamus revamp

### ISSUE THREAD

**Issue by giacecco** [theodi/shared#567]
Check that [Hippodamus](https://github.com/OpenAddressesUK/hippodamus) is still working vs all the changes of the last few months and publish one full set of up to date, downloadable datasets.

**Comment by pezholio** [comment]
Just tried to run this in Heroku, and I'm getting the error `Error R14 (Memory quota exceeded)`. As we're only (potentially) running this once a month, is it worth switching to a PX dyno? This costs $0.80 per hour, although we get 46 hours for free.

**Comment by giacecco** [comment]
I believe we should proceed unless there are obvious ways you can use to reduce memory usage. Have you got an idea of the time it's going to take, or how long it took the last time (if relevant, after the latest changes)? 

@JeniT @peterkwells any objections?

**Comment by pezholio** [comment]
I've split it up now, so we can run each part in parallel, but from memory, it took about a day, so if everything's running in parallel, I don't *think* we'll ever go over the free limit, but I can't say for sure

**Comment by pezholio** [comment]
Ran this yesterday, but it didn't upload because I had the wrong credentials. It took just over an hour though, and there's 8 separate tasks that should take the same time. If we run them every week (say, Sunday night), we should be comfortably within the free limit. Might be worth *not* running distillations on Sunday night, so we don't have any inconsistencies during the run.

**Comment by giacecco** [comment]
That's very good news.

[I'm ok with weekly hippodamus runs](https://github.com/theodi/shared/issues/566) and assuring consistency by not running distillations while that happens. Is it safe enough to manage that based on time only, or we need to find a more robust way, e.g. for one process to know about the other?

If it takes so little time, it also allow us to be smarter with https://github.com/theodi/shared/issues/568 as we can calculate the "expected URL" of new downloadable data client-side :-)

**Comment by pezholio** [comment]
I've kicked this off to do a test run. As we're running in parallel, it should take just over an hour. If all goes OK, I'll close this.

**Comment by pezholio** [comment]
I've also set distiller to NOT run on a Monday morning

**Comment by pezholio** [comment]
Right, so we can't do this in parallel, as MongoLab craps out, so I've set each task to run on the hour from 12am to 7am, which should space things out. Scheduling a test run for tomorrow morning, will see how that pans out.

**Comment by pezholio** [comment]
Closing this. See #566 for progress
